playground/NodeEditorLib.js

```diff
@@ -0,0 +1,134 @@
+import { NodePrototypeEditor } from './editors/NodePrototypeEditor.js';
+import { ScriptableEditor } from './editors/ScriptableEditor.js';
+import { BasicMaterialEditor } from './editors/BasicMaterialEditor.js';
+import { StandardMaterialEditor } from './editors/StandardMaterialEditor.js';
+import { PointsMaterialEditor } from './editors/PointsMaterialEditor.js';
+import { FloatEditor } from './editors/FloatEditor.js';
+import { Vector2Editor } from './editors/Vector2Editor.js';
+import { Vector3Editor } from './editors/Vector3Editor.js';
+import { Vector4Editor } from './editors/Vector4Editor.js';
+import { SliderEditor } from './editors/SliderEditor.js';
+import { ColorEditor } from './editors/ColorEditor.js';
+import { TextureEditor } from './editors/TextureEditor.js';
+import { UVEditor } from './editors/UVEditor.js';
+import { PreviewEditor } from './editors/PreviewEditor.js';
+import { TimerEditor } from './editors/TimerEditor.js';
+import { SplitEditor } from './editors/SplitEditor.js';
+import { JoinEditor } from './editors/JoinEditor.js';
+import { StringEditor } from './editors/StringEditor.js';
+import { FileEditor } from './editors/FileEditor.js';
+import { CustomNodeEditor } from './editors/CustomNodeEditor.js';
+
+export const ClassLib = {
```

Comment:
Maybe we can use here the same approach of `addSomething`?

Reply:
I don't think that's necessary at this point.